src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -166,15 +166,19 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
     });
   }
 
-  public autoResize(event): void {
-    const checkTextAreaHeight =
-      event.target.scrollHeight > this.textAreasHeight.minTextAreaScrollHeight &&
-      event.target.scrollHeight < this.textAreasHeight.maxTextAreaScrollHeight;
-    const maxHeight = checkTextAreaHeight
-      ? this.textAreasHeight.maxTextAreaHeight
-      : event.target.scrollHeight < this.textAreasHeight.minTextAreaScrollHeight;
-    const minHeight = checkTextAreaHeight ? this.textAreasHeight.minTextAreaHeight : `${event.target.scrollHeight}px`;
-    event.target.style.height = checkTextAreaHeight ? maxHeight : minHeight;
+  public autoResize(textarea: string, e: any) {
+    switch (textarea) {
+      case '1':
+        e.target.style.height = '0px';
+        e.target.style.height = e.target.scrollHeight + 'px';
+        break;
+      case '2':
+        e.target.style.height = '131px';
+        e.target.style.height = e.target.scrollHeight + 'px';
+        break;
+      default:
+        break;
+    }
```

Comment:
Switch case for 2 cases? Looks like a code smell.
Replace it with 'if else' condition